package-lock.json

```diff
@@ -507,6 +507,7 @@
             "integrity": "sha512-1Yj8h9Q+QDF5FzhMs/c9+6UntbD5MkRfRwac8DoEm9ZfUBZ7tZ55YcGVAzEe4bXsdQHEk+s9S5wsOKVdZrw0tQ==",
             "dev": true,
             "optional": true
+            "dev": true
```

Comment:
@yzhang-gh,
Seems that you broke `package-lock.json`.
It's best regenerate it by npm.

Reply:
It's already regenerated (05f8d4f).